cmd/todo_issue.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+var todoIssueCmd = &cobra.Command{
+	Use:              "issue",
+	Short:            "Add a Issue to Todo list",
```

Comment:
What do you think about adding a `Hidden: true` in here so this command doesn't show in the usage message?
In this way, we can keep this code and avoid users calling it.

Reply:
Good suggestion.  I'll add it although I'm worried we're going to forget about it when we do add support :)